drivers/sensor/ina23x/ina237.c

```diff
@@ -0,0 +1,305 @@
+/*
+ * Copyright 2021 Grinn
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT ti_ina237
+
+#include <logging/log.h>
+#include <drivers/sensor.h>
+#include "ina237.h"
+#include "ina23x_common.h"
+
+LOG_MODULE_REGISTER(INA237, CONFIG_SENSOR_LOG_LEVEL);
+
+/**
+ * @brief Macros for creating the INA237 calibration value
+ *        CALIB = ((8192 * current_lsb * rshunt) / 100)
+ *        NOTE: The 8192 value is a fixed value internal to the
+ *              INA237 that is used to ensure scaling is properly
+ *              maintained.
+ *
+ * @param current_lsb Value of the Current register LSB in milliamps
+ * @param rshunt Shunt resistor value in milliohms
+ */
+#define INA237_CALIB(current_lsb, rshunt) ((8192 * current_lsb * rshunt) / 100)
```

Comment:
Is it not necessary to have unsigned suffix for these?

Reply:
I don't see a particular reason to use it here.